.github/workflows/e2e-test.yml

```diff
@@ -94,3 +94,37 @@ jobs:
           wait-on: 'http://localhost:5173, http://localhost:6002, http://localhost:5001'
           spec: ./cypress/e2e/wgp/*.cy.js
           working-directory: ./test/cypress
+
+  client-and-user-portal:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        run: npm install
+        working-directory: ./server/assets-v1/templates
+      
+      - name: Install cypress
```

Comment:
I notice that you "Install Cypress". Daniel, "usues, 
```
- name: Run cypress tests
        uses: cypress-io/github-action@v6
```
Am I missing something? You also have this same action later.... Why doesn't he also install cypress?

Reply:
Yes, the cypress image should handle the run